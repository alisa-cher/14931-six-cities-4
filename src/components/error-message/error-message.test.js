import React from 'react';
import renderer from 'react-test-renderer';
import ErrorMessage from "./error-message.jsx";

it(`Error message correctly`, () => {
  const tree = renderer
    .create(
        <ErrorMessage
          className={`test`}
          tag={`span`}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
