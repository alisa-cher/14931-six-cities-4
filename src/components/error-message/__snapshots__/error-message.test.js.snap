// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error message correctly 1`] = `
<span
  className="test"
  style={
    Object {
      "color": "red",
      "display": "block",
      "paddingTop": "5px",
    }
  }
>
  Something went wrong. Please try again later.
</span>
`;
